src/libraries/System.Net.Quic/tests/FunctionalTests/QuicConnectionTests.cs

```diff
@@ -398,7 +398,7 @@ public async Task GetStreamCapacity_OpenCloseStreamCanceledIntoNegative_CountsCo
             };
 
             (QuicConnection clientConnection, QuicConnection serverConnection) = await CreateConnectedQuicConnection(clientOptions);
-            await streamsAvailableFired.WaitAsync().WaitAsync(PassingTestTimeout);
+            await streamsAvailableFired.WaitAsync(PassingTestTimeout);
```

Comment:
This will change the behavior from throwing after timeout to silently continuing (and probably asserting below), is this intended?

Reply:
You're right. It's not intended, good catch!

Reply:
Fixed.